ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -96,10 +135,21 @@ final class Authorizer(now: () => Instant, ledgerId: String, participantId: Stri
     }
 
   private def ongoingAuthorization[Res](scso: ServerCallStreamObserver[Res], claims: Claims) =
-    new OngoingAuthorizationObserver[Res](scso, claims, _.notExpired(now()), permissionDenied())
+    new OngoingAuthorizationObserver[Res](
+      scso,
+      claims,
+      _.notExpired(now()),
+      authorizationError => {
+        // Note: only put the claims in the context, as the request can be huge
+        newLoggingContext("Claims" -> claims.toString) { implicit logCtx =>
```

Comment:
1. If you want to keep the claims around, here I would recommend creating a function that takes the claims and returns a `Seq[(String, String)]` (passing it with a `:_ *). Note that everywhere else in we use lower-case keys for logging context.
2. I'm not entirely sure we should create a whole new logging context here (putting the entirety of the claims in it), as we only have a single line that already reports the exact error with the information as to what went wrong and how. Logging contexts are useful when you have a nested series of calls that need to enrich the information along the way. If you think we should have a logging context by the time we get here and enrich it with the claims information, I would recommend doing it up the call stack.

Reply:
> I'm not entirely sure we should create a whole new logging context here

The only reason to use the logging context here is to make sure the error uses the same formatting as all other log messages. For example, in case we want to make ContextualizedLogger write JSON objects.

Without the entire claims in the log line, there is indeed no reason to use a `ContextualizedLogger`.

Reply:
What do you think about tackling this with the rest of the story around using our contextual logging more? I think that we should create logging contexts when receiving a request and pushing them down the call stack to enrich them, but I would refrain from putting extra keys in the logging context only for formatting a single line.

Reply:
I'd rather we didn't just do string munging because our logging API makes it hard to add extra one-off key-value pairs. Strings just need parsing later.

If we find ourselves doing this a lot, let's make an abstraction that handles it.

Reply:
I have removed the extra key-value pair because, as @stefanobaghino-da mentioned, the important information is already in the error message. The class now uses a plain logger.

Reply:
Not sure why we want to throw away useful information, but I'm happy to ship this for now until we re-evaluate logging in a more holistic fashion.